research/gnn-survey/train.py

```diff
@@ -32,13 +32,15 @@
 flags.DEFINE_list('hidden_dim', [32], 'Dimension of gnn hidden layers')
 flags.DEFINE_enum('optimizer', 'adam', ['adam', 'sgd'],
                   'Optimizer for training')
+flags.DEFINE_integer('num_heads', 8, 'Number of multi-head attentions')
 flags.DEFINE_float('weight_decay', 5e-4, 'Weight for L2 regularization')
 flags.DEFINE_string('save_dir', 'models/cora/gcn',
                     'Directory stores trained model')
+flags.DEFINE_boolean('normalize_adj', True, 'Whether to normalize adj matrix')
+flags.DEFINE_boolean('sparse_features', True, 'Whether to use sparse features')
```

Comment:
This feature doesn't seem to entirely work. I couldn't get GAT to work with this and saved models with sparse tensors didn't load. This can be addressed later, though.